packages/@webex/plugin-meetings/test/unit/spec/networkQualityMonitor/index.js

```diff
@@ -42,12 +42,12 @@ describe('plugin-meetings', () => {
         roundTripTimeMeasurements: 14,
       },
       statsAnalyzerCurrentStats: {
-        audio: {
+        'video-send': {
```

Comment:
here "audio" was changed to "video-send" and few line further "video" to "audio-send" - can you double check if this is correct?

Reply:
I changed it back to maintain consistency. Tests still pass as the values changing weren't enough to change the test result.